### PR TITLE
Add new KPI block situations page VR

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard14**
Add KPI block on the new situations page on VR level.
This includes new Lokalize keys.

Waiting for the data to check before merging

### CODE REVIEW

**Review by VWSCoronaDashboard8 — APPROVED** [review]

File: packages/app/src/pages/veiligheidsregio/[code]/brononderzoek.tsx [hunk 1]
```diff
@@ -102,6 +114,58 @@ export default function BrononderzoekPage(
...
+                      investigations_total: (
+                        <InlineText color="data.primary">
+                          {formatNumber(mockData.investigations_total)}
+                        </InlineText>
+                      ),
+                    }
+                  )}
+                </Text>
+              </KpiTile>
+            ) : undefined}
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/pages/veiligheidsregio/[code]/brononderzoek.tsx:167]: React components that do not render anything should typically return null but maybe undefined works as well in this case... You could also use `&&` and then you don't need the else part I guess.

**Review by VWSCoronaDashboard8 — APPROVED** [review]

File: packages/app/src/pages/veiligheidsregio/[code]/brononderzoek.tsx [hunk 2]
```diff
@@ -64,6 +74,8 @@ export default function BrononderzoekPage(
     description: text.metadata.description,
   };
 
+  const mockData = mockKpiBlock();
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/pages/veiligheidsregio/[code]/brononderzoek.tsx:77]: please get rid of this and use data from the props (`data.situations.last_value`)

File: packages/app/src/pages/veiligheidsregio/[code]/brononderzoek.tsx [hunk 3]
```diff
@@ -102,6 +114,58 @@ export default function BrononderzoekPage(
           />
 
           <ArticleStrip articles={content.articles} />
+
+          <TwoKpiSection>
+            <Tile>Empty tile</Tile>
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/pages/veiligheidsregio/[code]/brononderzoek.tsx:119]: what's this?

File: packages/app/src/pages/veiligheidsregio/[code]/brononderzoek.tsx [hunk 4]
```diff
@@ -102,6 +114,58 @@ export default function BrononderzoekPage(
...
+            {mockData.has_sufficient_data ? (
+              <KpiTile
+                title={siteText.vr_brononderzoek.kpi_result.title}
+                metadata={{
+                  date: mockData.date_of_insertion_unix,
+                  source: text.bronnen.rivm,
+                }}
+              >
+                <KpiValue
+                  data-cy="covid_total"
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/pages/veiligheidsregio/[code]/brononderzoek.tsx:129]: copy-paste?

File: packages/app/src/pages/veiligheidsregio/[code]/brononderzoek.tsx [hunk 5]
```diff
@@ -102,6 +114,58 @@ export default function BrononderzoekPage(
...
+                  percentage={mockData.known_percentage}
+                />
+                <Text>
+                  {replaceComponentsInText(
+                    siteText.vr_brononderzoek.kpi_result.description,
+                    {
+                      date_start_unix: (
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/pages/veiligheidsregio/[code]/brononderzoek.tsx:136]: can you rename the variables to something without `_unix`? Also you can use the `intl.formatDateSpan` utility for formatting these dates.

**Review by VWSCoronaDashboard10 — CHANGES_REQUESTED** [review]

File: packages/app/src/pages/veiligheidsregio/[code]/brononderzoek.tsx (hunk 3 above)
Comment (VWSCoronaDashboard14) [reply packages/app/src/pages/veiligheidsregio/[code]/brononderzoek.tsx:119]: A placeholder for https://github.com/minvws/nl-covid19-data-dashboard/pull/2939

**Pull request merged**